CHANGES.rst

```diff
@@ -1,3 +1,17 @@
+1.16.1 (unreleased)
+===================
+
+outlier_detection
+-----------------
+
+- Decrease the amount of file I/O required to compute the median when `in_memory`
+  is set to `False`. [#8782]
+
+resample
+--------
+
+- Permit creating drizzled models one at a time in many-to-many mode. [#8782]
+
```

Comment:
```suggestion
```

now that #8671 is merged (switching change log handling to `towncrier`) this change log entry should be a file in `changes/` instead:
```shell
echo "Decrease the amount of file I/O required to compute the median when ``in_memory`` is set to ``False``." > changes/8782.outlier_detection.rst
echo "Permit creating drizzled models one at a time in many-to-many mode." > changes/8782.resample.rst
```

(new PRs will include instructions on how to do this)

Reply:
Thanks @zacharyburnett! I will put this in along with other pending review comments